lib/MusicBrainz/Server/Form/Utils.pm

```diff
@@ -231,6 +232,25 @@ sub indentation {
     return "\N{NO-BREAK SPACE}" x (3 * $level);
 }
 
+sub validate_username {
+    my ($self) = @_;
+
+    my $username = $self->value;
+    my $previous_username = $self->init_value;
+
+    if (defined $username) {
```

Comment:
That does sound reasonable, yeah... Wonder why the check was here originally then, given this was only used in the registration page!